### ISSUE TITLE
Piwik 2.14.0 queuedtracking:process error

### ISSUE THREAD

**Issue by andypl78** [piwik/plugin-QueuedTracking#15]
This is output manual running queuedtracking:process

./console queuedtracking:process -vvv

  [Piwik\Container\ContainerDoesNotExistException]
  The root container has not been created yet.

Exception trace:
 () at /var/www/sites.tld/web/core/Container/StaticContainer.php:40
 Piwik\Container\StaticContainer::getContainer() at /var/www/sites.tld/web/core/Container/StaticContainer.php:80
 Piwik\Container\StaticContainer::get() at /var/www/sites.tld/web/core/Access.php:88
 Piwik\Access::getInstance() at /var/www/sites.tld/web/core/Access.php:458
 Piwik\Access::doAsSuperUser() at /var/www/sites.tld/web/core/Console.php:80
 Piwik\Console->doRun() at /var/www/sites.tld/web/vendor/symfony/console/Symfony/Component/Console/Application.php:126
 Symfony\Component\Console\Application->run() at /var/www/sites.tld/web/console:27

**Comment by Jarla** [comment]
Hi, i had the same Problem and changed 2 things in 
Commands/Processor.php function execute and its working now

```php
protected function execute(InputInterface $input, OutputInterface $output)
    {
        // Workaround for 2.14.0 bug
        $environment = new \Piwik\Application\Environment('tracker');
        $environment->init();

        $systemCheck = new SystemCheck();
        $systemCheck->checkRedisIsInstalled();

        Access::getInstance()->setSuperUserAccess(false);
        Plugin\Manager::getInstance()->setTrackerPluginsNotToLoad(array('Provider'));
        Tracker::loadTrackerEnvironment();
        $this->recreateEagerCacheInstanceWhichChangesOnceTrackerModeIsEnabled();

        if (OutputInterface::VERBOSITY_VERY_VERBOSE <= $output->getVerbosity()) {
            Tracker::setTrackerDebugMode(true);
        }

        $backend      = Queue\Factory::makeBackend();
        $queueManager = Queue\Factory::makeQueueManager($backend);

        if (!$queueManager->canAcquireMoreLocks()) {
            $this->writeSuccessMessage($output, array("Nothing to proccess. Already max number of workers in process."));
            return;
        }

        $shouldProcess = false;
        foreach ($queueManager->getAllQueues() as $queue) {
            if ($queue->shouldProcess()) {
                $shouldProcess = true;
                break;
            }
        }

        if (!$shouldProcess) {
            $this->writeSuccessMessage($output, array("No queue currently needs processing"));
            return;
        }

        $output->writeln("<info>Starting to process request sets, this can take a while</info>");

        register_shutdown_function(function () use ($queueManager) {
            $queueManager->unlock();
        });

        $startTime = microtime(true);
        $processor = new Processor($queueManager);
        $processor->setNumberOfMaxBatchesToProcess(1000);
        $workaround_tracker = new Tracker($queueManager);
        $tracker = $processor->process($workaround_tracker);

$neededTime = (microtime(true) - $startTime);
        $numRequestsTracked = $tracker->getCountOfLoggedRequests();
        $requestsPerSecond  = $this->getNumberOfRequestsPerSecond($numRequestsTracked, $neededTime);

        Piwik::postEvent('Tracker.end');

        $this->writeSuccessMessage($output, array(sprintf('This worker finished queue processing with %sreq/s (%s requests in %02.2f seconds)', $requestsPerSecond, $numRequestsTracked, $neededTime)));
    }
```

**Comment by andypl78** [comment]
After making changes no longer displays the error but also does not update page visits.

./console queuedtracking:monitor
Queue is enabled
Request sets in the queue will be processed automatically after a tracking request
Up to 1 workers will be used
Processor will start once there are at least 10 request sets in the queue
20515 (20515) request sets left in queue. 74.44M used memory (274.89M peak). 0 workers active.

**Comment by tsteur** [comment]
It should be fixed now and available in latest release 0.1.7 which will be available soon. Please comment or reopen in case it doesn't work for you. It worked for me.
